WPFSKillTree/Model/Items/EquipmentData.cs

```diff
@@ -11,19 +11,20 @@ namespace POESKillTree.Model.Items
 {
     public class EquipmentData
     {
-        private readonly Options _options;
 
         public IReadOnlyDictionary<ItemType, IReadOnlyList<Affix>> AffixesPerItemType { get; private set; }
 
         public IReadOnlyList<ItemBase> BaseList { get; private set; }
 
         public IReadOnlyDictionary<string, ItemBase> BaseDictionary { get; private set; }
 
+        public ItemImageService ItemImageService { get; }
```

Comment:
I didn't even know this was allowed by the language... Maybe an actual accessor function + private variable would be less... confusing here? (I totally get how it's consequential in its current form)